test/python/algorithms/test_measure_error_mitigation.py

```diff
@@ -80,10 +90,11 @@ def test_measurement_error_mitigation_with_diff_qubit_order(self):
 
         self.assertRaises(QiskitError, quantum_instance.execute, [qc1, qc3])
 
-    def test_measurement_error_mitigation_with_vqe(self):
+    @data("CompleteMeasFitter", "TensoredMeasFitter")
+    def test_measurement_error_mitigation_with_vqe(self, fitter_str):
         """measurement error mitigation test with vqe"""
         try:
-            from qiskit.ignis.mitigation.measurement import CompleteMeasFitter
+            from qiskit.ignis.mitigation.measurement import CompleteMeasFitter, TensoredMeasFitter
             from qiskit import Aer
             from qiskit.providers.aer import noise
```

Comment:
As we have this try-except block multiple times it might be nice to move it to the top of the file and just store a `_HAS_IGNIS` variable or which we can use to skip the tests.

Reply:
Change made